heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -192,6 +194,12 @@ bool StMgrClient::SendTupleStreamMessage(proto::stmgr::TupleStreamMessage& _msg)
   if (IsConnected()) {
     SendMessage(_msg);
```

Comment:
Just to confirm. The drop upon backpressure logic is handled in the "(!IsConnected())" branch. Is it expected? Do we need it in the "(IsConnected)" branch?

Reply:
Redid the logic